strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -829,7 +826,7 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
                 tf.convert_to_tensor(meas_result * np.sqrt(m_omega_over_hbar / 2))
             )
             quad_eigenstate = ops.displacement(
-                displacement_size, 0, inf_squeezed_vac, self._cutoff_dim, True, self._batched
+                tf.math.abs(displacement_size), tf.math.angle(displacement_size), 0, inf_squeezed_vac, self._cutoff_dim, True, self._batched
```

Comment:
(Minor suggestion): defining `r_d` and `phi_d` before this call and including them here